crates/flox/doc/flox-init.md

```diff
@@ -12,19 +12,22 @@ flox-init - initialize flox expressions for current project.
 # SYNOPSIS
 
 flox [ `<general-options>` ] init [ `<options>` ]
-# DESCRIPTION
 
-Add a new package using a template.
+# DESCRIPTION
 
-Given a `<template>` creates a new package definition in `PROJECT_ROOT/<name>`.
+Create a new empty environment in the current directory.
 
-An existing package called `<name>` will raise an error.
+The name of the environment will be the basename of the current directory
+or "default" if the current directory is `$HOME`.
+The `--name` flag can be used to give the environment a custom name.
 
-If `<template>` or `<name>` are unspecified and a controlling TTY is present,
-flox will query them using interactive dialogs.
+By default the environment will be created in the current directory.
+Flox will add a directory `$PWD/.flox`,
+within which all relevant metadata of the environment will be tracked.
```

Comment:
question nonblocking: should we say something about what should be committed to version control?

Reply:
I think we should do that once the layout and files are finalized